### DISCUSSION TITLE
Potential issue with PimcoreSimpleBackendSearchBundle

### DISCUSSION THREAD

**Discussion by RaederDev** [pimcore/pimcore#16226] [Q&A / Support]
I am currently in the middle of updating my Pimcore install from 10.x to 11.x. I think I found a bug but am unsure if it's not expected behaviour so I decided to ask here first.

After following all steps in the upgrade guide I was missing the Search functionality. I added PimcoreSimpleBackendSearchBundle::class to my bundles.php like instructed but it still did not show up.

Then I discovered this discussion which solved the problem:
https://github.com/pimcore/pimcore/discussions/15984

There are a bunch of migrations in the CoreBundle like for example Version20221215071650.php that automatically migrate existing installed versions of the GlossaryBundle, SeoBundle and a whole bunch of others and those all seem to work fine without running the bundle install command. I was unable to find a migration that did the same for the search bundle so I assume this is causing the issue of having to manually run the bundle install command after upgrading pimcore. What is also confusing to me is why I need to run the install command if I already installed it with the bundles.php file.

Having to maintain installed bundles in the new settings store is a huge pain for setups that have multiple environments, is there a way to configure pimcore to store that info in a yaml file that can be committed instead?

**Comment by dvesh3** [comment]
Adding bundle entry to `bundles.php` only registers the bundle (Symfony way) and enables it for installation. Most of the bundles still need to call the installer as they perform some operations required for proper functioning of the feature introduced by the bundle e.g. adding user permissions, creating db tables and so on. The migrations on 10.6 marks those bundles as already installed since they were extracted from the core and data exists for these bundles so you just need to enable them in the upgrade process and you are good to go.

Unfortunately, there's no other way to check these bundles are installed than settings-store.
